mods/ts/rules/shared-support.yaml

```diff
@@ -39,3 +42,31 @@ NAPULS:
 		FactionImages:
 			gdi: napuls.gdi
 			nod: napuls.nod
+	ProvidesPrerequisite@gdi:
+		Prerequisite: emp.gdi
+		Factions: gdi
+		ResetOnOwnerChange: true
+	ProvidesPrerequisite@nod:
+		Prerequisite: emp.nod
+		Factions: nod
+		ResetOnOwnerChange: true
+	AttackOrderPower@nod:
+		Cursor: emp
+		Icon: emp
+		ChargeTime: 135
+		Description: E.M. Pulse
+		LongDesc: Fires a pulse blast which disables\nall mechanical units in the area.
+		EndChargeSound: speech-nod|00-i158.aud
```

Comment:
You should be able to push these out to faction-specific voice variants, like we do in d2k.